package.json

```
@@ -11,6 +11,9 @@
     "lint:fix": "eslint --fix . && prettier --write .",
     "test": "run-s build:* && run-s test:*",
     "test:e2e": "xvfb-maybe cross-env NODE_ENV=test playwright test 'test/e2e/.*.test.js'",
+    "vtestpreview": "vite preview",
+    "vtest": "vitest",
+    "vtest:ui": "vitest --ui",
```

```suggestion
    "vitestpreview": "vite preview",
    "vitest": "vitest",
    "vitest:ui": "vitest --ui",
```

Looking at the vitest docs, it doesn't look like calling it "vtest" is a comon pattern, or is it?

---

Also, these scripts fail currently, because there are no tests yet. Could you please add a dummy test, so that we can verify the setup is working properly?

---

Agree